setup.py

```diff
@@ -6,13 +6,14 @@
 
 data_dir = os.path.join(sysconfig.get_python_lib(), "hamster", "data")
 
-setup(name='hamster-sqlite',
-      version='0.3',
-      description='Minimal dependency nicely abstracted sqlite backend of hamster time tracker - lets you connect to your hamster db and do stuff in python',
-      author='Toms Baugis',
-      author_email='toms.baugis@gmail.com',
-      url='https://github.com/projecthamster/hamster',
-      package_dir = {'': 'src'},
-      py_modules = ['hamster.storage', 'hamster.db', 'hamster.lib.__init__'],
-      data_files=[(data_dir, ['data/hamster.db'])],
-     )
+setup(
```

Comment:
Much better without the quote replacement, but I see little value in this indentation change either.